semcore/button/src/component/ButtonLink/ButtonLink.tsx

```diff
@@ -12,6 +12,8 @@ const enhance = {
   resolveColor: resolveColorEnhance(),
 };
 
+// TODO: find how Button becames visually-disabled={disabled}
```

Comment:
My bad, `visually-disabled` comes from `Link` styles. Your fix is correct, but you don't need this comment

Reply:
Fixed.